packages/yasqe/grammar/sparql11-grammar.pl

```diff
@@ -804,6 +852,10 @@
 % e.g. DOUBLE, DECIMAL, INTEGER
 % e.g. INTEGER_POSITIVE, PLUS
 tm_punct([
+'<<'= '<<',
+'>>'= '>>',
+'{|'= '\\{\\|',
+'|}'= '\\|\\}',
```

Comment:
Here's where I struggled the most. I hadn't realized I had to escape the right elements. I suspected I needed to move the `<<` before other punctuation like `<` and `<=`, but it escaped me to escape it :grimacing: The error message was a CodeMirror error, saying the stream could not be processed. Took me at least one hour, then a walk outside, to come back and review everything and seeing other examples here I escaped these and everything worked fine :+1: :tada: